app/main.py

```diff
@@ -16,4 +16,14 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+variables = [
+    lucky_number, pi, one_is_a_prime_number, name, my_favourite_films,
+    profile_info, marks, collection_of_coins]
+mutable_obj = ("list", "dict", "set")
+sorted_variables = {"mutable": [], "immutable": []}
+
+for var_def in variables:
```

Comment:
No need to have functionality to place variables by their types.
Just place them manually